src/repository/crop.repository.ts

```diff
@@ -0,0 +1,26 @@
+import { uuidv4 } from "@firebase/util";
+import { CropRequestBody } from "@/models/crop/CropRequestBody";
+import prisma from "../../prisma/client";
+
+export const isExistCrop = async (userId: string) => {
+  const isExist = await prisma.crops.findMany({
+    where: {
+      user_id: userId,
+      harvested_at: null,
+    },
+  });
+  return isExist.length != 0;
+};
```

Comment:
아래와 같이 명명하는게 더 명확할것같아요.
```suggestion
export const isExistGrowingCrop = async (userId: string) => {
  const growingCrops = await prisma.crops.findMany({
    where: {
      user_id: userId,
      harvested_at: null,
    },
  });
  return growingCrops.length !== 0;
};
```